pkg/db/db.go

```diff
@@ -53,6 +53,10 @@ type Operation interface {
 	PutRedHatCPEs(tx *bolt.Tx, cpeIndex int, cpe string) (err error)
 	RedHatRepoToCPEs(repository string) (cpeIndices []int, err error)
 	RedHatNVRToCPEs(nvr string) (cpeIndices []int, err error)
+	//for K8s
+	GetK8sOutdatedAPI(key string) (outdatedapi types.OutDatedAPIData, err error)
+	PutK8sOutdatedAPI(tx *bolt.Tx, key string, apis interface{}) (err error)
```

Comment:
```suggestion
	GetK8sDb(key string) (outdatedapi types.OutDatedAPIData, err error)
	PutK8sDb(tx *bolt.Tx, key string, apis interface{}) (err error)
```
What if we create "universal" name for these functions?
In the future, we will want to use these functions for other k8s databases.